code/game/objects/effects/anomalies/_anomalies.dm

```diff
@@ -41,10 +41,7 @@
 		aSignal.code = rand(1,100)
 		aSignal.anomaly_type = type
 
-		var/frequency = rand(MIN_FREE_FREQ, MAX_FREE_FREQ)
-		if(ISMULTIPLE(frequency, 2))//signaller frequencies are always uneven!
```

Comment:
Is the behavior of frequencies always being uneven maintained?

Reply:
Yes but it's automatic within the sanitize_frequency function that he changed it to